### ISSUE TITLE
[ENHANCEMENT] Make nick/user list pinnable

### ISSUE THREAD

**Issue by geisesteert** [MCMrARM/revolution-irc#113]
- as with the channel list, make the user/nick list pin'able
- alternatively, add config option to always expand it

Reasoning: On desktop UI's, such as Chromebooks, and probably tablets in landscape mode, there is a lot of virtually unused screen estate (used only for very long messages), while the current list of users is not visible and needs to be swiped into the screen manually each time.

Attached example screenshots.

![screenshot 2018-03-07 at 02 10 06](https://user-images.githubusercontent.com/21268333/37066469-38ae8e08-21ae-11e8-9c92-3e3716ee6e1f.png)

![screenshot 2018-03-07 at 02 09 47](https://user-images.githubusercontent.com/21268333/37066452-22f0c874-21ae-11e8-8bca-f08e3873dbbd.png)

**Comment by MCMrARM** [comment]
This possibly will not really look great in the next update, but it should work with the upcoming design somehow.

**Comment by MCMrARM** [comment]
Nope, I can't come up with a good UX for this unfortunately, moving to the 0.4 milestone (which may end up being 0.5 as I'm considering renaming 0.3.3 to 0.4).
